network/kademlia_test.go

```diff
@@ -461,23 +456,43 @@ func TestSuggestPeers(t *testing.T) {
 	tk.checkSuggestPeer("<nil>", 0, false)
 }
 
+//Tests change of saturationDepth returned by suggestPeers
 func TestSuggestPeersSaturationDepthChange(t *testing.T) {
 
 	base := "00000000"
 	tk := newTestKademlia(t, base)
 	tk.On("00100000", "00010000", "10000000", "11000000", "11100000", "01000000", "01100000")
 
 	//Saturation depth is 2
+	if tk.saturationDepth != 2 {
+		t.Fatalf("Saturation depth should be 2, got %d", tk.saturationDepth)
+	}
 	tk.Off("01000000")
 	//Saturation depth should have fallen to 1
 	tk.checkSuggestPeer("01000000", 1, true)
 	tk.On("01000000")
 
 	//Saturation depth is 2 again
+	if tk.saturationDepth != 2 {
+		t.Fatalf("Saturation depth should be 2, got %d", tk.saturationDepth)
+	}
 	tk.Off("10000000")
 	//Saturation depth should have fallen to 0
```

Comment:
check?

Reply:
You mean the saturation is 0? if so, it's checked by the checkSugestPeer helper function